### ISSUE TITLE
Add decentralised proof of person 

### ISSUE THREAD

**Issue by ubiubi18** [twitter/the-algorithm#414]
To stop bots confusing the algoritm twitter should consider doing some deeper research on cryptoidentity and maybe even borrow some existing proof of person mechanism. 

It is possible, look for example at simultanous flipsessions of idena https://www.idena.io/flip-challenge

**Comment by ConnieCommie** [comment]
Do you fart

**Comment by dumbuz** [comment]
Yeah, this could allow people who don't have $8/mo to spend to be considered "people" on this hellscape and not be pushed into the void by the algorithm.

**Comment by cs3c** [comment]
![giphy](https://user-images.githubusercontent.com/48242964/229245833-e6e86303-610f-487f-a345-61615094a6be.gif)

**Comment by samdisk11** [comment]
> To stop bots confusing the algoritm twitter should consider doing some deeper research on cryptoidentity and maybe even borrow some existing proof of person mechanism.
> 
> It is possible, look for example at simultanous flipsessions of idena https://www.idena.io/flip-challenge

Random question: would that flip challenge work without JavaScript?

**Comment by ubiubi18** [comment]
all essential parts are written in go  https://github.com/idena-network/idena-go

**Comment by samdisk11** [comment]
> all essential parts are written in go https://github.com/idena-network/idena-go

Oh... i see, its not a captcha API as i thought.

**Comment by darkdevildeath** [comment]
In my opinion, the responsibility and competence for identifying individuals should be the responsibility of the government of each country. And Twitter should just indicate this diplomatically in its regular meetings with state officials.
It is insecure and unsustainable to leave this in the hands of those who create online systems. And the only solution that guarantees accuracy involves physical verification.
Where each person appears in front of a public official with certified documents and then is inserted into an open-source API used by the state for software to use and be able to validate.
I even have a model that I presented to parliamentarians in my country for this purpose.
My model even allows identifying the source of personal data leaks.

**Comment by dumbuz** [comment]
> In my opinion, the responsibility and competence for identifying individuals should be the responsibility of the government of each country. And Twitter should just indicate this diplomatically in its regular meetings with state officials. It is insecure and unsustainable to leave this in the hands of those who create online systems. And the only solution that guarantees accuracy involves physical verification. Where each person appears in front of a public official with certified documents and then is inserted into an open-source API used by the state for software to use and be able to validate. I even have a model that I presented to parliamentarians in my country for this purpose. My model even allows identifying the source of personal data leaks.

Anonymity is good. Dont take this away from us.
